grpc/cubic.proto

```diff
@@ -201,6 +201,12 @@ message RecognitionConfig {
 
     // μ-law (8-bit) encoded RAW data, single channel, sampled at 8 KHz.
     ULAW8000 = 5;
+
+    // A-law (8-bit) encoded RAW data, single channel, sampled at 8 KHz.
+    ALAW8000 = 6;
+
+    // Opus (16-bit) encoded RAW data
```

Comment:
Does Opus specify its sample rate the way WAV or FLAC does? If so, should we use similar wording in the comment?
```suggestion
    // Opus (16-bit) encoded RAW data, sampled at a rate equal to or higher than the sample rate expected by the chosen Model.
```